src/server/game/Spells/SpellMgr.cpp

```diff
@@ -3276,17 +3253,15 @@ void SpellMgr::LoadSpellInfoCorrections()
         46008, // Negative Energy
         45641, // Fire Bloom
         55665, // Life Drain - Sapphiron (H)
-        28796, // Poison Bolt Volly - Faerlina
-        41303, // Soul Drain (Reliquary of Souls)
-        40243, // Crushing Shadows (Teron Gorefiend)
-        30657, // Quake (Magtheridon)
```

Comment:
Why these were removed?

Reply:
moved out to respective scripts, re @shuaren